contracts/PolygonZkEVM.sol

```diff
@@ -1187,10 +1221,9 @@ contract PolygonZkEVM is OwnableUpgradeable, EmergencyManager {
     function setVerifyBatchTimeTarget(
         uint64 newVerifyBatchTimeTarget
     ) external onlyAdmin {
-        require(
-            newVerifyBatchTimeTarget < 1 days,
-            "PolygonZkEVM::setVerifyBatchTimeTarget: setVerifyBatchTimeTarget incorrect range"
-        );
+        if (newVerifyBatchTimeTarget > 1 days) {
```

Comment:
Is this a conscious choice to save a little bit of gas by not doing `newVerifyBatchTimeTarget >= 1 days`?

Reply:
difference here would be negligible as already discussed in #22.

Reply:
Yes! it is!